controllers/salons.js

```diff
@@ -428,4 +432,42 @@ exports.deleteRating = async (req, res, next) => {
     } catch (e) {
         next(e)
     }
+}
+
+exports.serviceDetails = async (req, res, next) => {
```

Comment:
What does this method do? If I'm not wrong, is it creating a service? If so, it shouldn't be in the `business` endpoint where is whole logic about managing salons?